index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/4da7da78a5.js" crossorigin="anonymous"></script>
+    <link href="https://cdn.lineicons.com/3.0/lineicons.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
I think that it would be better to use only 2 spaces as indentations instead of 4 spaces which is the default space for the tab key (You can change the default tab space size in your VSCode settings).

Reply:
I actually have 2 space Indentations already set for the tab key so I think it is a result of how my document was formatted. I will make changes to that as well. Thanks for bringing my attention to it 😄.